test/unit/ajax.js

```diff
@@ -527,6 +527,23 @@ QUnit.module( "ajax", {
 		};
 	} );
 
+	ajaxTest( "jQuery.ajax() - timeout", 2, function( assert ) {
```

Comment:
`timeout` -> `native timeout`, similarly to how the `native abort` test case is named.